index.html

```diff
@@ -193,6 +193,59 @@
 
 <p>This can be pretty useful for when you want to use <code>bind-show</code> and start in a visible state.</p>
 
+<p><code>context</code> lets you change the current context for every element nested inside.</p>
+
+<div class="prepend-pre">
+<div define="{object: {value: 'foo'} }" context="foo">
+  <span bind="value"></span> <!-- foo.value from inside the context of foo -->
+</div>
+<div>
+  <span bind="foo.value"> <!-- foo.value from outside the context of foo -->
+</div>
+</div>
+
+<h2>Define-array</h2>
+<p><code>define-array</code> lets you define an array of objects one at a time in the context and implicitly reference the correct member in the array.</p>
+
+<div class="prepend-pre">
+<div data-define-array="{key: {value: 5, foo: function(){ return 12; }}}">
```

Comment:
The name `key` is kind of confusing here. It made me think that it was a keyword with some kind of special behaviour related to `define-array` when really its just the name of the array you're appending to. I prefer names like `widgets` or `things` in docs like this.